src/config/ConfigReader.py

```diff
@@ -19,7 +20,8 @@
     "azure_general": "AzureApi",
     "azure_graph": "AzureGraph",
     "azure_mail_reports": "AzureMailReports",
-    "cloudflare": "Cloudflare"
+    "cloudflare": "Cloudflare",
+    "1password": "OnePassword"
```

Comment:
Why we have 2 naming conventions here?

Reply:
@yotamloe class name can't start with a number so `1password` is invalid name, but for users I think it would be more convenient to use `1password` as the API name instead of `onepassword` 🤔

Reply:
Okay, let's keep it that way since its also the commercial way they introduce their company. But let's align all other naming conventions in file names and tests files and names.